corehq/form_processor/models.py

```diff
@@ -258,9 +259,7 @@ class AbstractAttachment(models.Model):
 
     @property
     def filepath(self):
-        if getattr(settings, 'IS_TRAVIS', False):
-            return os.path.join('/home/travis/', str(self.attachment_id))
-        return os.path.join('/tmp/', str(self.attachment_id))
+        return os.path.join(gettempdir(), str(self.attachment_id))
```

Comment:
yeah `/tmp` is not writeable unfortunately (pretty odd). i didn't do too much digging to try and find a way around it. potentially could do something with sudo, but the new container based systems require you to have `sudo: false`. not entirely sure what that option restricts though